_includes/footer.html

```diff
@@ -6,9 +6,9 @@
                 <p>Made with ❤️ by the Vanilla OS Contributors.</p>
             </div>
             <div class="footer-area">
-                <p>Vanilla OS is a Free and Open Source project. <a href="{{ site.url }}/get-involved">Learn here</a> how to get involved.</p>
+                <p>Vanilla OS is a free and open-source project. <a href="{{ site.url }}/get-involved">Learn here</a> how to get involved.</p>
                 <p>
-                    <small>Every software and component is licensed under different Open Source licenses.</small>
+                    <small>Every component and piece of software is distributed under open-source licenses.</small>
```

Comment:
At the same time, not everything in Vanilla OS *is* under the same license though, so saying that it's under "a" license is wrong

Reply:
Keep in mind that this also refers to Ubuntu/Debian itself

Reply:
"An open-source license" doesn't say it's the same license, it just means everything has one, it can be any open-source license.

Reply:
It really does sound like it's referring to a single license, mostly because from a language perspective it **is**

Reply:
I still don't understand how "an" implies a single thing, take this as an example:
All my friends have cars.
All my friends have a car.

Does the second sentence not sound correct?

Reply:
It's more like "all my friends are in a car"

Reply:
I agree with @MonsterObserver that it should be "under an open-source license". The subject isn't Vanilla OS, but "Every software and component", which has a degree of separation. If the subject was Vanilla OS, then "under open-source licenses" would make more sense, because it's one entity. Since we are talking about multiple things under one group, it makes more sense to say that each software is under an open-source license.

> It's more like "all my friends are in a car"

This could also imply that, no matter the situation, your friends are inside a car, whether they are in the same car or not. Contrarily, if you said "all my friends are in the car", it means that they're all in the same car.

That being said, I would take a step further and write "Every component and piece of software is distributed under various free and open-source licenses." I think it sounds more natural.

Reply:
Honestly, to avoid complicating our lives, what about "Every component and piece of software is free and open-source."?

Reply:
Sounds fine to me